<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="mvnProject" duration-ms="11701" started-at="2018-11-27T00:13:33Z" finished-at="2018-11-27T00:13:45Z">
    <groups>
    </groups>
    <test name="mvnProjectTest" duration-ms="11701" started-at="2018-11-27T00:13:33Z" finished-at="2018-11-27T00:13:45Z">
      <class name="com.mvnProject.testcases.LoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.mvnProject.testcases.LoginPageTest@3f0ee7cb]" name="setUp" is-config="true" duration-ms="9684" started-at="2018-11-26T18:13:33Z" finished-at="2018-11-26T18:13:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="LoginPageTitleTest()[pri:0, instance:com.mvnProject.testcases.LoginPageTest@3f0ee7cb]" name="LoginPageTitleTest" duration-ms="22" started-at="2018-11-26T18:13:43Z" finished-at="2018-11-26T18:13:43Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.mvnProject.testcases.LoginPageTest.LoginPageTitleTest(LoginPageTest.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPageTitleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.mvnProject.testcases.LoginPageTest@3f0ee7cb]" name="tearDown" is-config="true" duration-ms="1166" started-at="2018-11-26T18:13:44Z" finished-at="2018-11-26T18:13:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.mvnProject.testcases.LoginPageTest -->
    </test> <!-- mvnProjectTest -->
  </suite> <!-- mvnProject -->
</testng-results>
